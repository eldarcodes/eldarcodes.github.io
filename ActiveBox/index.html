<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link href="https://fonts.googleapis.com/css?family=Cardo:400i|Open+Sans:700|Raleway:300,400,600,700&display=swap"
        rel="stylesheet">
    <link rel="stylesheet" href="style.css">
    <title>ActiveBox</title>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
</head>

<body>
    <!-- SVG -->
    <svg style="display: none;">
        <symbol id="facebook" viewBox="0 0 96.124 96.123">
            <g>
                <path
                    d="M72.089,0.02L59.624,0C45.62,0,36.57,9.285,36.57,23.656v10.907H24.037c-1.083,0-1.96,0.878-1.96,1.961v15.803
                            c0,1.083,0.878,1.96,1.96,1.96h12.533v39.876c0,1.083,0.877,1.96,1.96,1.96h16.352c1.083,0,1.96-0.878,1.96-1.96V54.287h14.654
                            c1.083,0,1.96-0.877,1.96-1.96l0.006-15.803c0-0.52-0.207-1.018-0.574-1.386c-0.367-0.368-0.867-0.575-1.387-0.575H56.842v-9.246
                            c0-4.444,1.059-6.7,6.848-6.7l8.397-0.003c1.082,0,1.959-0.878,1.959-1.96V1.98C74.046,0.899,73.17,0.022,72.089,0.02z" />
            </g>
        </symbol>
        <symbol id="twitter" viewBox="0 -47 512.00203 512">
            <path
                d="m191.011719 419.042969c-22.140625 0-44.929688-1.792969-67.855469-5.386719-40.378906-6.335938-81.253906-27.457031-92.820312-33.78125l-30.335938-16.585938 32.84375-10.800781c35.902344-11.804687 57.742188-19.128906 84.777344-30.597656-27.070313-13.109375-47.933594-36.691406-57.976563-67.175781l-7.640625-23.195313 6.265625.957031c-5.941406-5.988281-10.632812-12.066406-14.269531-17.59375-12.933594-19.644531-19.78125-43.648437-18.324219-64.21875l1.4375-20.246093 12.121094 4.695312c-5.113281-9.65625-8.808594-19.96875-10.980469-30.777343-5.292968-26.359376-.863281-54.363282 12.476563-78.851563l10.558593-19.382813 14.121094 16.960938c44.660156 53.648438 101.226563 85.472656 168.363282 94.789062-2.742188-18.902343-.6875-37.144531 6.113281-53.496093 7.917969-19.039063 22.003906-35.183594 40.722656-46.691407 20.789063-12.777343 46-18.96875 70.988281-17.433593 26.511719 1.628906 50.582032 11.5625 69.699219 28.746093 9.335937-2.425781 16.214844-5.015624 25.511719-8.515624 5.59375-2.105469 11.9375-4.496094 19.875-7.230469l29.25-10.078125-19.074219 54.476562c1.257813-.105468 2.554687-.195312 3.910156-.253906l31.234375-1.414062-18.460937 25.230468c-1.058594 1.445313-1.328125 1.855469-1.703125 2.421875-1.488282 2.242188-3.339844 5.03125-28.679688 38.867188-6.34375 8.472656-9.511718 19.507812-8.921875 31.078125 2.246094 43.96875-3.148437 83.75-16.042969 118.234375-12.195312 32.625-31.09375 60.617187-56.164062 83.199219-31.023438 27.9375-70.582031 47.066406-117.582031 56.847656-23.054688 4.796875-47.8125 7.203125-73.4375 7.203125zm0 0" />
        </symbol>
        <symbol id="linkedin" viewBox="0 0 430.117 430.118">
            <g>
                <path id="LinkedIn__x28_alt_x29_" d="M398.355,0H31.782C14.229,0,0.002,13.793,0.002,30.817v368.471
                            c0,17.025,14.232,30.83,31.78,30.83h366.573c17.549,0,31.76-13.814,31.76-30.83V30.817C430.115,13.798,415.904,0,398.355,0z
                             M130.4,360.038H65.413V165.845H130.4V360.038z M97.913,139.315h-0.437c-21.793,0-35.92-14.904-35.92-33.563
                            c0-19.035,14.542-33.535,36.767-33.535c22.227,0,35.899,14.496,36.331,33.535C134.654,124.415,120.555,139.315,97.913,139.315z
                             M364.659,360.038h-64.966V256.138c0-26.107-9.413-43.921-32.907-43.921c-17.973,0-28.642,12.018-33.327,23.621
                            c-1.736,4.144-2.166,9.94-2.166,15.728v108.468h-64.954c0,0,0.85-175.979,0-194.192h64.964v27.531
                            c8.624-13.229,24.035-32.1,58.534-32.1c42.76,0,74.822,27.739,74.822,87.414V360.038z M230.883,193.99
                            c0.111-0.182,0.266-0.401,0.42-0.614v0.614H230.883z" />
            </g>
        </symbol>
    </svg>


    <!-- header -->
    <header class="header" id="header">
        <div class="container">
            <div class="header__inner">
                <div class="header__logo">
                    <img src="img/ActiveBox_logo.png" alt="">
                </div>
                <nav class="nav">
                    <a href="#" class="nav__link"data-scroll="#features">Features</a>
                    <a href="#" class="nav__link"data-scroll="#works">Works</a>
                    <a href="#" class="nav__link"data-scroll="#team">Our Team</a>
                    <a href="#" class="nav__link"data-scroll="#reviews">Testimonials</a>
                    <a href="#" class="nav__link"data-scroll="#download">Download</a>
                </nav>
            </div>
        </div>
    </header>

    <!-- intro -->
    <div class="intro" id="intro">
        <div class="container">
            <div class="intro__inner">
                <h1 class="intro__title">Your Favorite One Page
                    Multi Purpose Template</h1>
                <h2 class="intro__subtitle">
                    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Aut explicabo natus voluptatum, odit quis
                    aliquam quasi ea recusandae, obcaeca
                </h2>
                <a href="#" class="btn btn--red">Find out more</a>
            </div>
        </div>
    </div>

    <!-- Features -->
    <div class="container" id="features">
        <div class="features">
            <div class="features__item">
                <img src="img/benefits/benefits1.png" alt="" class="features__icon">
                <h4 class="features__title">Easily Customised</h4>
                <div class="features__texts">Cras justo odio, dapibus ac facilisis in, egestas
                    eget quam. Nullam quis risus eget urna mollis ornare vel eu leo.
                    Donec ullamcorper nulla non metus auctor </div>
            </div>
            <div class="features__item">
                <img src="img/benefits/benefits2.png" alt="" class="features__icon">
                <h4 class="features__title">Responsive REady</h4>
                <div class="features__texts">Cras justo odio, dapibus ac facilisis in, egestas
                    eget quam. Nullam quis risus eget urna mollis ornare vel eu leo.
                    Donec ullamcorper nulla non metus auctor </div>
            </div>
            <div class="features__item">
                <img src="img/benefits/benefits3.png" alt="" class="features__icon">
                <h4 class="features__title">Modern Design</h4>
                <div class="features__texts">Cras justo odio, dapibus ac facilisis in, egestas
                    eget quam. Nullam quis risus eget urna mollis ornare vel eu leo.
                    Donec ullamcorper nulla non metus auctor </div>
            </div>
            <div class="features__item">
                <img src="img/benefits/benefits4.png" alt="" class="features__icon">
                <h4 class="features__title">Clean Code</h4>
                <div class="features__texts">Cras justo odio, dapibus ac facilisis in, egestas
                    eget quam. Nullam quis risus eget urna mollis ornare vel eu leo.
                    Donec ullamcorper nulla non metus auctor </div>
            </div>
            <div class="features__item">
                <img src="img/benefits/benefits5.png" alt="" class="features__icon">
                <h4 class="features__title">Ready to ship</h4>
                <div class="features__texts">Cras justo odio, dapibus ac facilisis in, egestas
                    eget quam. Nullam quis risus eget urna mollis ornare vel eu leo.
                    Donec ullamcorper nulla non metus auctor </div>
            </div>
            <div class="features__item">
                <img src="img/benefits/benefits6.png" alt="" class="features__icon">
                <h4 class="features__title">Download for free</h4>
                <div class="features__texts">Cras justo odio, dapibus ac facilisis in, egestas
                    eget quam. Nullam quis risus eget urna mollis ornare vel eu leo.
                    Donec ullamcorper nulla non metus auctor </div>
            </div> <!-- features /  -->
        </div> <!-- features /  -->
    </div> <!-- container /  -->

    <!-- works -->

    <div class="works" id="works">
        <div class="works__item">
            <img class="works__photo" src="img/works/work-1.jpg" alt="">
            <div class="works__content">
                <div class="works__title">Eldar project</div>
                <div class="works__text">User interface design</div>
            </div>
        </div>
        <div class="works__item">
            <img class="works__photo" src="img/works/work-2.jpg" alt="">
            <div class="works__content">
                <div class="works__title">Eldar project</div>
                <div class="works__text">User interface design</div>
            </div>
        </div>
        <div class="works__item">
            <img class="works__photo" src="img/works/work-3.jpg" alt="">
            <div class="works__content">
                <div class="works__title">Eldar project</div>
                <div class="works__text">User interface design</div>
            </div>
        </div>
        <div class="works__item">
            <img class="works__photo" src="img/works/work-4.jpg" alt="">
            <div class="works__content">
                <div class="works__title">Eldar project </div>
                <div class="works__text">User interface design</div>
            </div>
        </div>
        <div class="works__item">
            <img class="works__photo" src="img/works/work-5.jpg" alt="">
            <div class="works__content">
                <div class="works__title">Eldar project</div>
                <div class="works__text">User interface design</div>
            </div>
        </div>
        <div class="works__item">
            <img class="works__photo" src="img/works/work-6.jpg" alt="">
            <div class="works__content">
                <div class="works__title">Eldar project</div>
                <div class="works__text">User interface design</div>
            </div>
        </div>
        <div class="works__item">
            <img class="works__photo" src="img/works/work-7.jpg" alt="">
            <div class="works__content">
                <div class="works__title">Eldar project</div>
                <div class="works__text">User interface design</div>
            </div>
        </div>
        <div class="works__item">
            <img class="works__photo" src="img/works/work-3.jpg" alt="">
            <div class="works__content">
                <div class="works__title">Eldar project</div>
                <div class="works__text">User interface design</div>
            </div>
        </div>
    </div> <!-- /works  -->

    <!-- Team -->
    <div class="team" id="team">
        <div class="container">
            <div class="team__inner">
                <div class="team__item">
                    <img class="team__photo" src="img/team/team-1.jpg" alt="">
                    <div class="team__name">Ruth Woods</div>
                    <div class="team__prof">Founder, CEO</div>
                    <div class="team__text">
                        <p>Fusce dapibus, tellus ac cursus commodo, tortor mauris
                            condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed
                            diam eget risus varius blandit sit amet non magna.
                            Nullam quis risus eget urna mollis ornare vel eu leo.</p>
                    </div>
                    <div class="social">
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#facebook">
                                </use>
                            </svg>
                        </a>
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#twitter">
                                </use>
                            </svg>
                        </a>
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#linkedin">
                                </use>
                            </svg>
                        </a>
                    </div>
                </div>
                <div class="team__item">
                    <img class="team__photo" src="img/team/team-2.jpg" alt="">
                    <div class="team__name">Timothy Reed</div>
                    <div class="team__prof">Co-Founder, Developer</div>
                    <div class="team__text">
                        <p>Fusce dapibus, tellus ac cursus commodo, tortor mauris
                            condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed
                            diam eget risus varius blandit sit amet non magna.
                            Nullam quis risus eget urna mollis ornare vel eu leo.</p>
                    </div>
                    <div class="social">
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#facebook">
                                </use>
                            </svg>
                        </a>
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#twitter">
                                </use>
                            </svg>
                        </a>
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#linkedin">
                                </use>
                            </svg>
                        </a>
                    </div>
                </div>
                <div class="team__item">
                    <img class="team__photo" src="img/team/team-3.jpg" alt="">
                    <div class="team__name">Victoria Valdez</div>
                    <div class="team__prof">UI Designer</div>
                    <div class="team__text">
                        <p>Fusce dapibus, tellus ac cursus commodo, tortor mauris
                            condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed
                            diam eget risus varius blandit sit amet non magna.
                            Nullam quis risus eget urna mollis ornare vel eu leo.</p>
                    </div>
                    <div class="social">
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#facebook">
                                </use>
                            </svg>
                        </a>
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#twitter">
                                </use>
                            </svg>
                        </a>
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#linkedin">
                                </use>
                            </svg>
                        </a>
                    </div>
                </div>
                <div class="team__item">
                    <img class="team__photo" src="img/team/team-4.jpg" alt="">
                    <div class="team__name">Beverly Little</div>
                    <div class="team__prof">Data Scientist</div>
                    <div class="team__text">
                        <p>Fusce dapibus, tellus ac cursus commodo, tortor mauris
                            condimentum nibh, ut fermentum massa justo sit amet risus. Maecenas sed
                            diam eget risus varius blandit sit amet non magna.
                            Nullam quis risus eget urna mollis ornare vel eu leo.</p>
                    </div>
                    <div class="social">
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#facebook">
                                </use>
                            </svg>
                        </a>
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#twitter">
                                </use>
                            </svg>
                        </a>
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#linkedin">
                                </use>
                            </svg>
                        </a>
                    </div>
                </div>
            </div> <!-- /team__inner -->
        </div> <!-- /container -->
    </div> <!-- /team -->

    <!-- Reviews -->
    <div class="reviews" id="reviews">
        <div class="container">
            <div class="reviews__item">
                <div class="reviews__photo">
                    <img class="reviews__img" src="img/reviews/review-1.jpg" alt="">
                </div>

                <div class="reviews__content">
                    <div class="reviews__text">“Integer posuere erat a ante venenatis dapibus
                        posuere velit aliquet. Donec sed odio dui. Aenean eu leo quam...”</div>
                    <div class="reviews__author">Susan Sims, Interaction Designer at XYZ</div>
                </div>
            </div>
        </div>
    </div> <!-- / Reviews -->

    <!-- download -->
    <div class="download" id="download">
        <div class="container">
            <h3 class="download__title">Are You Ready to Start? Download Now For Free!</h3>
            <div class="download__text">
                <p>Fusce dapibus, tellus ac cursus commodo</p>
            </div>
            <a href="#" class="btn btn--long btn--red">Download Here</a>
        </div> <!-- /container -->
    </div> <!-- /download -->

    <!-- footer -->
    <footer class="footer">
        <div class="container">
            <div class="footer__inner">
                <div class="footer__block">
                    <h4 class="footer__title">Location</h4>
                    <address class="footer__address">
                        <p>3481 Melrose Place</p>
                        <p>Beverly Hills, CA 90210</p>
                    </address>
                </div>
                <div class="footer__block">
                    <h4 class="footer__title">Share with love</h4>
                    <div class="social social--footer">
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#facebook">
                                </use>
                            </svg>
                        </a>
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#twitter">
                                </use>
                            </svg>
                        </a>
                        <a href="#" class="social__item" target="_blank">
                            <svg class="social__icon">
                                <use xlink:href="#linkedin">
                                </use>
                            </svg>
                        </a>
                    </div>
                </div>
                <div class="footer__block">
                    <h4 class="footer__title">About ActiveBox</h4>
                    <div class="footer__text">Cras justo odio, dapibus ac facilisis in, egestas eget quam. Donec
                        ullamcorper nulla non metus auctor fringilla.</div>
                </div>
            </div> <!-- /footer__inner -->
        </div> <!-- /container -->
        <div class="copyright">
            <div class="container">
                <div class="copyright__text">
                    <div>Copyright © 2015 ActiveBox. All Rights Reserved/div>
                        <div>Made <span>by Eldar Mirzabekov</span></div>
                    </div>
                </div>
            </div>
    </footer> <!-- /footer -->

<script src="app.js"></script>


</body>

</html>