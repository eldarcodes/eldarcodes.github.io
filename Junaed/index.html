<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="style.css">
    <title>Junaed</title>
    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700,800&display=swap" rel="stylesheet">
</head>

<body>
    <!-- header -->
    <header class="header">
        <div class="container">
            <div class="header_inner">
                <nav class="header_nav">
                    <a href="#" class="nav_link">HOME</a>
                    <a href="#" class="nav_link">PORTFOLIO</a>
                    <a href="#" class="nav_link">ABOUT</a>
                    <a href="#" class="nav_link">CONTACT</a>
                </nav>
            </div> <!-- /.header_inner -->
        </div> <!-- /.container -->
    </header> <!-- /.header -->


    <!-- intro -->
    <div class="intro">
        <div class="container">
            <div class="intro_inner">
                <div class="intro_left">
                    <div class="intro_title">
                        our strong organaisation
                    </div>
                    <div class="intro_text">
                        Lorem ipsum dolor sit amet, consectetur adipis cing elit, sed do eiusmod tempor incididunt ut
                        labore et. ectetur adig ipis cing elit, sed do eiusmod tempor incididunt.
                    </div>
                    <a href="#" class="intro_btn">
                        Contact Us
                    </a>
                </div>
                <div class="intro_right">
                    <!-- <div class="video">
                        <img class="video__icon" src="img/Vedio Icon.png" alt="">
                    </div> -->
                    <iframe class="video" width="560" height="315" src="https://www.youtube.com/embed/8vTzF_TrSsw"
                        frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
                        allowfullscreen></iframe>
                </div>
            </div> <!-- /.intro_inner -->
        </div> <!-- /.container -->
    </div> <!-- /.intro -->


    <!-- services -->
    <div class="services">
        <div class="container">
            <div class="services__inner">
                <div class="services__inner-left">
                    <div class="services__title">do you know we can
                        provide for you ?</div>
                    <div class="services__text">Lorem ipsum dolor sit amet, consectetur adipis cing elit, sed do eiusmod
                        tempor incididunt ut labore et.Lorem ni ipsum dolor sit amet, consectetur adipis cing elit, ed
                        doi eiusmod tempor incididunt ut labore et.</div>
                    <a href="#" class="services__button">Learn More</a>
                </div>
                <div class="services__inner-right">
                    <div class="services__item">
                        <div class="services__item-icon">
                            <img src="img/Settings.png" alt="">
                        </div>
                        <div class="services__item-title">Management</div>
                        <div class="services__item-text">
                            Lorem ipsum dolor sitea amet, zixf conseit adipi cing elit, seddi do eiusmod btdempor in
                            didunt utlae ore etioe.Lorem ipsum new idolor
                        </div>
                    </div>
                    <div class="services__item">
                        <div class="services__item-icon">
                            <img src="img/Pencil.png" alt="">
                        </div>
                        <div class="services__item-title">UI /UX Design</div>
                        <div class="services__item-text">
                            Lorem ipsum dolor sitea amet, zixf conseit adipi cing elit, seddi do eiusmod btdempor in
                            didunt utlae ore etioe.Lorem ipsum new idolor
                        </div>
                    </div>
                    <div class="services__item">
                        <div class="services__item-icon">
                            <img src="img/Diamond.png" alt="">
                        </div>
                        <div class="services__item-title">Logo / Branding</div>
                        <div class="services__item-text">
                            Lorem ipsum dolor sitea amet, zixf conseit adipi cing elit, seddi do eiusmod btdempor in
                            didunt utlae ore etioe.Lorem ipsum new idolor
                        </div>
                    </div>
                    <div class="services__item">
                        <div class="services__item-icon">
                            <img src="img/Truck 2.png" alt="">
                        </div>
                        <div class="services__item-title">Animation</div>
                        <div class="services__item-text">
                            Lorem ipsum dolor sitea amet, zixf conseit adipi cing elit, seddi do eiusmod btdempor in
                            didunt utlae ore etioe.Lorem ipsum new idolor
                        </div>
                    </div>
                </div>
            </div> <!-- /.services__inner -->
            <div class="line"></div>
        </div> <!-- /.container -->
    </div> <!-- /.services -->

    <!-- feature -->
    <div class="feature">
        <div class="container">
            <div class="feature__inner">
                <div class="feature__title">feature products</div>
                <div class="feature__text">
                    <p>
                        Lorem ipsum dolor sit amet, consectetur adipis cing elit,sed <br> do eiusmod tempor incididunt
                        ut labore et.
                    </p>
                </div>
                <div class="feature__line"></div>
                <div class="category">
                    <a href="#" class="category--link">ALL</a>
                    <a href="#" class="category--link">print template</a>
                    <a href="#" class="category--link">web template</a>
                    <a href="#" class="category--link">user interface</a>
                    <a href="#" class="category--link">mock-up</a>
                </div>
                <div class="category__images">
                    <img src="https://placehold.it/250" alt="" class="category__images-link">
                    <img src="https://placehold.it/250" alt="" class="category__images-link">
                    <img src="https://placehold.it/250" alt="" class="category__images-link">
                    <img src="https://placehold.it/250" alt="" class="category__images-link">
                    <img src="https://placehold.it/250" alt="" class="category__images-link">
                    <img src="https://placehold.it/250" alt="" class="category__images-link">
                    <img src="https://placehold.it/250" alt="" class="category__images-link">
                    <img src="https://placehold.it/250" alt="" class="category__images-link">
                </div>
            </div> <!-- /.feature__inner -->
        </div> <!-- /.container -->
    </div> <!-- /.feature -->

    <!-- team -->
    <div class="team">
        <div class="container">
            <div class="team__inner">
                <div class="team__title">Meet our team</div>
                <div class="team__text">
                    <p>Lorem ipsum dolor sit amet, consectetur adipis cing elit, sed dooing eiusmod tempor incididut
                        labore Ui / Ux , print template.</p>
                </div>
                <div class="team__content">
                    <img src="https://placehold.it/320x460" alt="" class="team__content-img">
                    <div class="team__content-right">
                        <div class="team__content-title">mark waugh</div>
                        <div class="team__content-subtitle">
                            <p>Lorem ipsum dolor sit amet, consectetur adipis cing elit, sed dooing eiusmod tempor
                                incididut labore Ui / Ux , print template.</p>
                        </div>
                        <div class="team__content-social">
                            <a class="team__content-social-link" href="#">Facebook</a>
                            <a class="team__content-social-link" href="#">Dribble</a>
                            <a class="team__content-social-link" href="#">Behance</a>
                            <a class="team__content-social-link" href="#">Twitter</a>
                        </div>
                        <div class="team__works-images">
                            <div class="img__wrap">
                                <img src="https://placehold.it/200x215" alt="" class="team__content-works">
                                <p class="img__description">SOKINA JOE</p>
                            </div>
                            <div class="img__wrap">
                                <img src="https://placehold.it/200x215" alt="" class="team__content-works">
                                <p class="img__description">SOKINA JOE</p>
                            </div>
                            <div class="img__wrap">
                                <img src="https://placehold.it/200x215" alt="" class="team__content-works">
                                <p class="img__description">SOKINA JOE</p>
                            </div>
                            <div class="img__wrap">
                                <img src="https://placehold.it/200x215" alt="" class="team__content-works">
                                <p class="img__description">SOKINA JOE</p>
                            </div>
                        </div>
                    </div>
                </div>
            </div> <!-- /.team__inner -->
        </div> <!-- /.container -->
    </div> <!-- /.team -->

    <!-- subscribe -->
    <div class="subscribe">
        <div class="subscribe__inner">
            <div class="subscribe__title">Design tips, tricks, and freebies. Delivered weekly.</div>
            <div class="subscribe__subtitle">Lorem ipsum dolor sit amet, consectetur adipis cing elit, sed do eiusmod.
            </div>
            <form method="POST" class="subscribe__form" action="/">
                <input class="input" type="text" placeholder="Email address...">
                <button class="btn-red">
                    <a href="#">SUBSCRIBE</a>
                </button>
            </form>
        </div>
    </div>

</body>

</html>