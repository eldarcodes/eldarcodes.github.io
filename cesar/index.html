<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Шифр Цезаря</title>
    <link rel="stylesheet" href="style.css">
</head>

<body>
    <div class="container">
        <h1 class="title">Caesar cipher</h1>
        <div class="left">
            <h3 class="subtitle">Шифровать текст:</h3>
            <input class="input" type="text" id="in">
            <p id="out">Result:</p>
        </div>
        <div class="right">
            <h3 class="subtitle">Расшифровать текст:</h3>
            <input class="input" type="text" id="out2">
            <p id="out3">Result: </p>
        </div>
        <footer>
            <div class="copyright">
                &copy; by mirzabekov228
            </div>
        </footer>

    </div>

    <script src="script.js"></script>
</body>

</html>