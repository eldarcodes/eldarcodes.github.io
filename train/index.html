<!DOCTYPE html>
<html>

<head>
    <meta charset="utf-8">
    <title>Typing Test</title>
    <link rel="stylesheet" type="text/css" href="style.css">
    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=PT+Sans+Narrow" rel="stylesheet">
</head>

<body id="body">
    <div class="animated fadeInUp" id="main">
        <div id="mistakes"></div>
        <div id="alertText"></div>
        <div id="field"><span id="text"></span></div>
    </div>
    <script type="text/javascript" src="main.js"></script>
</body>

</html>