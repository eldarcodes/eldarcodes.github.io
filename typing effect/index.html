<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Typing Effect</title>
    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="style.css">
</head>

<body>
    <div class="container">
        <p>Леха это <span class="typed-text"></span><span class="cursor">&nbsp;</span></p>
    </div>

    <script src="main.js"></script>
</body>

</html>