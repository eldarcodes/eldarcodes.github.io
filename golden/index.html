<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="css/style.css">
    <title>Golden</title>
</head>

<body>

    <!-- header -->
    <header class="header">
        <div class="container">
            <div class="header__inner">
                <div class="header__logo">
                    <a href="#">
                        <img src="img/header-logo.png" alt="">
                    </a>
                </div>
                <nav class="header__menu">
                    <a class="header__menu-link" href="#">Home</a>
                    <a class="header__menu-link" href="#">Services</a>
                    <a class="header__menu-link" href="#">Portfolio</a>
                    <a class="header__menu-link" href="#">About</a>
                    <a class="header__menu-link" href="#">Contact</a>
                </nav>
            </div>
            <!-- /.header__inner -->
        </div>
        <!-- /.container -->
    </header>
    <!-- /.header -->

    <!-- intro -->
    <div class="intro">
        <div class="container">
            <div class="intro__inner">
                <div class="intro__subtitle">Welcome To Our Studio!</div>
                <div class="intro__title">Itâ€™s nice to Meet you</div>
                <a href="#" class="intro__btn">Tell Me More</a>
            </div>
        </div>
        <!-- /.container -->
    </div>
    <!-- /.intro -->

    <!-- Services -->
    <div class="services">
        <div class="container">
            <div class="services__inner">
                <div class="services__title">Services</div>
                <div class="services__subtitle">Proin iaculis purus consequat sem cure. </div>
                <div class="services__column">
                    <div class="services__item">
                        <img class="services__item-logo" src="img/icon1.png" alt="">
                        <h3 class="services__item-title">E-Commerce</h3>
                        <p class="services__item-text">
                            Proin iaculis purus consequat sem cure
                            digni ssim. Donec porttitora entum suscipit
                            aenean rhoncus posuere odio in tincidunt.
                        </p>
                    </div>
                    <div class="services__item">
                        <img class="services__item-logo" src="img/icon2.png" alt="">
                        <h3 class="services__item-title">Responsive Web</h3>
                        <p class="services__item-text">
                            Proin iaculis purus consequat sem cure
                            digni ssim. Donec porttitora entum suscipit
                            aenean rhoncus posuere odio in tincidunt.
                        </p>
                    </div>
                    <div class="services__item">
                        <img class="services__item-logo" src="img/icon3.png" alt="">
                        <h3 class="services__item-title">Web Security</h3>
                        <p class="services__item-text">
                            Proin iaculis purus consequat sem cure
                            digni ssim. Donec porttitora entum suscipit
                            aenean rhoncus posuere odio in tincidunt.
                        </p>
                    </div>
                </div> <!-- /.services__column -->
            </div> <!-- /.services__inner -->
        </div> <!-- /.container -->
    </div> <!-- /.Services -->

    <!-- portfolio -->

    <div class="portfolio">
        <div class="container">
            <div class="portfolio__inner">
                <div class="portfolio__title">Our portfolio</div>
                <div class="portfolio__subtitle">Proin iaculis purus consequat sem cure. </div>
                <div class="portfolio__items">
                    <img src="img/img1.png" alt="" class="portfolio__items-image">
                    <img src="img/img2.png" alt="" class="portfolio__items-image">
                    <img src="img/img3.png" alt="" class="portfolio__items-image">
                </div>
            </div> <!-- /.portfolio__inner -->
        </div> <!-- /.container -->
    </div> <!-- /.portfolio -->

    <!-- About -->
    <div class="about">
        <div class="container">
            <div class="about__inner">
                <div class="about__title">About Us</div>
                <div class="about__subtitle">Proin iaculis purus consequat sem cure. </div>
                <div class="about__info">
                    <div class="about__item-left">
                        <div class="about__content">
                            <div class="about__content-date">July 2010</div>
                            <div class="about__content-title">Our Humble Beginnings</div>
                            <div class="about__content-text">
                                <p>
                                    Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni
                                    ssim. Donec porttitora entum suscipit.
                                </p>
                            </div>
                        </div>
                        <img class="about__logo" src="img/about1.png" alt="">
                    </div>
                    <div class="about__item-right">
                        <img class="about__logo" src="img/about2.png" alt="">
                        <div class="about__content">
                            <div class="about__content-date">January 2011</div>
                            <div class="about__content-title">Facing Startup Battles</div>
                            <div class="about__content-text">
                                <p>
                                    Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni
                                    ssim. Donec porttitora entum suscipit.
                                </p>
                            </div>
                        </div>
                    </div>
                    <div class="about__item-left">
                        <div class="about__content">
                            <div class="about__content-date">December 2012</div>
                            <div class="about__content-title">Enter The Dark Days</div>
                            <div class="about__content-text">
                                <p>
                                    Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni
                                    ssim. Donec porttitora entum suscipit.
                                </p>
                            </div>
                        </div>
                        <img class="about__logo" src="img/about3.png" alt="">
                    </div>
                    <div class="about__item-right">
                        <img class="about__logo" src="img/about4.png" alt="">
                        <div class="about__content">
                            <div class="about__content-date">february 2014</div>
                            <div class="about__content-title">Our Triumph</div>
                            <div class="about__content-text">
                                <p>
                                    Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni
                                    ssim. Donec porttitora entum suscipit.
                                </p>
                            </div>
                        </div>
                    </div>
                    <div class="copyright">
                        <p>
                            &copy; Copyright 2014 FreebiesXpress.com
                        </p>
                    </div>
                </div> <!-- /.about__info -->
            </div> <!-- /.about__inner -->
        </div> <!-- /.container -->
    </div> <!-- /.about -->

</body>

</html>